tests/phpunit/tests/test-default-term.php

```diff
@@ -139,4 +139,14 @@ function test_default_category_in_list_table() {
 		$this->assertFalse( strpos( $list, 'edit-tags.php?action=delete&amp;taxonomy=category&amp;tag_ID=' . $fr . '&amp;' ) );
 		$this->assertNotFalse( strpos( $list, 'edit-tags.php?action=delete&amp;taxonomy=category&amp;tag_ID=' . $id . '&amp;' ) );
 	}
+
+	function test_get_option_default_category() {
+		$this->pll_admin->pref_lang = self::$model->get_language( 'es' );
+		$this->pll_admin->set_current_language();
```

Comment:
This line seems useless to me.
In fact we never have the condition to set the current language in this test and we always fall in the `pll_no_language_defined` case.

See https://github.com/polylang/polylang/blob/master/admin/admin-base.php#L339-L377